lib/order_query/nulls_direction.rb

```diff
@@ -32,7 +32,7 @@ def parse!(direction)
     # @return [:first, :last] the default nulls order, based on the given
     #   scope's connection adapter name.
     def default(scope, dir)
-      case scope.connection_config[:adapter]
+      case scope.connection.adapter_name
```

Comment:
Does this establish a connection? The previous version didn't though I'm not sure if this makes any difference.

Reply:
Hi! Actually I am not sure if it establishes a connection. How to check?

Reply:
```suggestion
      case scope.connection_db_config.adapter
```

`connection.adapter_name` does establish the connection, but using `connection_db_config.adapter` does not.

https://github.com/rails/rails/pull/38005